### ISSUE TITLE
It seems the moving circle marker is out of order again...

### ISSUE THREAD

**Issue by CptHolzschnauz** [pR0Ps/grafana-trackmap-panel#67]
With  Grafana v8.0.6 the moving circle marker is not working anymore.

**Comment by drecchia** [comment]
Same here. +1

**Comment by umbm** [comment]
same here and no mousover action displayed

**Comment by leonardopivetta** [comment]
looking at the code it seems like you are using the old angular way of getting the cursor position, that's not compatible with the new Timeseries panel.

After some research, I found that the react solution for getting the cursor position is to subscribe to the `eventbus `with the `DataHoveEvent`. 
 
Hope this will help the developers 😁

**Comment by Bart-1992** [comment]
I'm having the same issue, hope the developer can find some time to solve this for the new Timeseries panel :)

**Comment by karthikhkumar** [comment]
I too have similar issue.  This missing feature helps in the correlation of different panels. It would be awesome if this issue is fixed

**Comment by chaos-user** [comment]
Hi everyone.
I'm facing the same issue about dot point drawing. 
@leonardopivetta is possbile to share a workaround (please) to fix this issue until developers publish a new release (please) of this plugin?
Without dot drawing related to data timeline navigation, this amazing plugin lose its usability.

**Comment by chaos-user** [comment]
Hi everyone.

I found a "workaround" to keep "dot functionality" by keep using old Graph at Grafana instead of time series graph.


![image](https://user-images.githubusercontent.com/75700201/188605707-1903bbfb-5ff7-4796-a9a9-2c1dde3baf74.jpg)

![image2](https://user-images.githubusercontent.com/75700201/188605553-0730c766-a073-4c2d-b255-85aab76a4e50.jpg)


**NOTE:** This is not a final solution or even recommended. It's just a way to keep track plugin compatibility, but using an old and deprecated Grafana plugin

**Comment by CptHolzschnauz** [comment]
As mentioned before: It's only working with the Graph(old) tile, NOT with the new Time Series tile. 
I don't think Graph(Old) is not deprecated, at least works fine for me with no issues or missing features.

**Comment by chaos-user** [comment]
@CptHolzschnauz , you are right. The track plugin keeps working with Graph(old) panel from grafana.
My point is Graph panel have been "replaced" by time-series panel since Grafana 7.0 when they update and unify panels architecture and performance.
https://grafana.com/blog/2021/02/10/how-the-new-time-series-panel-brings-major-performance-improvements-and-new-visualization-features-to-grafana-7.4/

By default at a fresh new grafana installation, the Graph old panel is not available, only Time-Series....but we can "edit" JSON and change timeseries to graph to make old graph panel available again :)

Is for that reason that I mentioned old Graph has been deprecated, but you are right... it still works as before.
